src/newton_raphson.toit

```diff
@@ -85,12 +85,12 @@ solve --initial/num=0.0 --goal/num=0.0 --max-iterations=20 --precision=1e9 [--fu
       return old-diff.abs < new-diff.abs ? x : new-x
     previous = x
     x = new-x
-  no-convergence.call x
+  no_convergence.call x
```

Comment:
Here.

Reply:
we could use 'no-convergence' here, but since we must use `no_convergence` at the declaration point that would feel inconsistent.